Final Project/Editor Interface/Set-up-Instructions.md

```diff
@@ -6,9 +6,15 @@ Follow these instructions in order to run editor interface and hop stars:
 
 - Download the folder **Editor Interface** in your PC.
 - The script is written in Python with backend in JavaScript and a python compiler is needed to run the file. So, ensure that you've a python compiler in your PC.
-- In our project, we're using all the 4 csv files inside [Database](https://github.com/Liza23/Star-Hopping-KSP/tree/master/Final%20Project/Editor%20Interface/Database) folder. So, download/ add the database folder in the same **Editor Interface** folder where you're working.  
-- You must also download [requirements.txt](https://github.com/SahyadriDK/Hopping/blob/master/requirements.txt) file in the same folder, if not already present. 
-- You can use your own database as well, but this database used here is the very exhaustively compiled and is compatible with the code. In case you want to use your own database, ensure that you change the code accordingly.
+
+## Downloading the Database.
+
+- Within the editor interface folder lies the **Database** folder. This is where all the database csv files, once downloaded, will be placed.
+- This folder contains three files - IAU-CSN.csv which is the file containing star names,`query_program_input.py` which is the program that will download the database, and requirements.txt.
```

Comment:
i'll have to change the database folder as well right?
It should now have 3 files as you mentioned?

Reply:
Yeah. Sorry that I didnt mention that earlier. The three files are up in the main repo that Chaitanya maintains. The folder is aptly called Database. So you can pull from there.